auth/login.js

```diff
@@ -0,0 +1,61 @@
+import { button, inputEmail, inputPassword, emailError, pwError, showError, hiddenError} from "./common.js";
+
+// email 유효성 검사 패턴
+const pattern = /^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,}$/;
+
+// email 유효성 검사 및 에러 메세지
+const isValidEmail = () => {
+  const emailValue = inputEmail.value.trim(); // 이메일 입력값을 공백 제거 후 가져오기
+
+  if (emailValue === "") {
+    showError(inputEmail, emailError, "이메일을 입력해주세요.")
+    return false;
+  } else if (!pattern.test(emailValue)) { // 정규식으로 이메일 형식 검사하기
+    showError(inputEmail, emailError, "잘못된 이메일 형식입니다.");
+    return false;
+  } else {
+    hiddenError(inputEmail, emailError);
+    return true;
+  }
```

Comment:
❗️ 수정요청
early return 을 하고 계시기 때문에 else if문으로 사용하지 않고 아래처럼 작성하시는 것이 동일동작하며 가독성측면에서도 더 유리합니다~

```suggestion
    return false;
  }
  
  if (!pattern.test(emailValue)) {
    showError(inputEmail, emailError, "잘못된 이메일 형식입니다.");
    return false;
  }
  
  hiddenError(inputEmail, emailError);
  return true;
```